content/cloud/custom-domains-tutorial.md

```diff
@@ -0,0 +1,71 @@
+
+## Prerequisites
+
+To ensure the tutorial goes smoothly, please ensure you have the following: 
+
+* Most importantly, you must have a domain you own. If you don't have a domain, please visit a domain registrar like [Namecheap](https://www.namecheap.com/) or [Google Domains](https://domains.google/?gad=1&gclid=Cj0KCQjwzdOlBhCNARIsAPMwjbyNg3a3GcrQjbs3vBSDg4AWnnEkc96mJupyg8nqqiKyxVCyaePRXfAaAjRLEALw_wcB&gclsrc=aw.ds) to purchase one. 
+* A Fermyon Cloud account that is set up via your preferred [GitHub user account](https://docs.github.com/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/remembering-your-github-username-or-email)
+* A Spin application you would like to apply the custom domain to. If you aren't familiar with how to deploy an application on Fermyon Cloud, check out our [develop a Spin application](./develop.md) article to learn how to build a Spin application and then our [deploy a Spin application article](./deploy.md) for deployment instructions. 
+
+## Select Your Spin Application
+
+In the [Fermyon Cloud Dashboard](https://cloud.fermyon.com), navigate to your Spin application of choice and click the "Manage Domains" button.
+
+![DRAFT SCREENSHOT - app panel view](/static/image/custom-domains/app-panel-view.png)
+
+## Input Your Custom Domain
+
+Select the "Add custom domain" button and input your desired domain name. 
+
+![DRAFT SCREENSHOT - step 1](/static/image/custom-domains/Step-1.png)
+
+
+Please add the complete domain name (i.e., `www.example.com`) as Fermyon Cloud does not support redirects at this time. Click save to confirm you would like to add this domain name to Fermyon Cloud.
+
+![DRAFT SCREENSHOT - step 2](/static/image/custom-domains/Step-2.png)
+
+Up next, you'll need to configure your DNS records with your registrar to point at Fermyon's name servers (NS). 
+
+>>If you click "I'll do this later" you will be pausing the workflow and your *custom domain will not work yet* with your Spin application.
+
+![DRAFT SCREENSHOT - step 3](/static/image/custom-domains/Step-3.png)
+
+## Update Your Domain Registrar With Fermyon Cloud Name Servers
+
+If you've never delegated this domain to Fermyon Cloud before, you'll proceed strate to updating your nameserver records with your domain registrar with the nameservers generated by Fermyon Cloud. Each domain registrar has it's own unqiue set of instructions for completing this workflow. Please visit your domain registrar's documentation for instructions. 
+
+![DRAFT SCREENSHOT - step 4](/static/image/custom-domains/Step-4.png)
+
+In addition to generating NS records, Fermyon Cloud will generate a Let's Encrypt certificate on your behalf. 
```

Comment:
This slightly mixes two kinds of "generate" - there's some stuff on the screen that the reader needs to go take to their registrar, and an internal that FC stashes away.  Like reading this I wasn't sure if I needed to do anything with the cert?

I'd consider removing the mention of the cert here, and when all is set up, remark that FC has generated (and will keep renewing) a cert for you.  Or say something like: "Note that you do _not_ need to provide a SSL certificate.  Fermyon Cloud creates one internally, and will keep renewing it for the lifetime of your application."  But I guess that applies equally to the "repeat delegation" case?

Reply:
I created a dedicated "callout" about cert management! I can move it all the way to the bottom if you think that would make more sense, but give it a read! :) I agree, the way it was previously written was confusing

Reply:
Feels great to have that sentence about Fermyon taking care of SSL renewal for the life of the app. That's a huge deal.
This is a GREAT value add.